installing/installing_aws/ipi/installing-aws-customizations.adoc

```diff
@@ -48,7 +48,7 @@ include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
 
 include::modules/installation-aws-arm-tested-machine-types.adoc[leveloffset=+2]
 
-include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-aws-config-yaml-customizations.adoc[leveloffset=+2]
 
```

Comment:
Since you're referring them to another assembly from the new module, I think it would help to add a link like this wherever it appears:
```suggestion
include::modules/installation-aws-config-yaml-customizations.adoc[leveloffset=+2]

[role="_additional-resources"]
.Additional resources

* xref:../../../installing/installing_aws/installation-config-parameters-aws.adoc#installation-config-parameters-aws[Installation configuration parameters for AWS]
```
(I didn't check my work on that xref :stuck_out_tongue:)

Reply:
Good idea, done